src/components/NewRuntimeModal.js

```diff
@@ -652,7 +683,7 @@ export const NewRuntimeModal = withModalDrawer({ width: 675, 'aria-labelledby':
         _.map(({ cost, label, unitLabel }) => {
           return div({ key: label, style: { flex: 1, ...styles.label } }, [
             div({ style: { fontSize: 10 } }, [label]),
-            div({ style: { color: colors.accent(1.1), marginTop: '0.25rem' } }, [
+            div({ style: { color: colors.accent(), marginTop: '0.25rem' } }, [
```

Comment:
Do we know why it's been explicitly set to `1.1`?

Reply:
Hmm... I'm not sure... I'll revert this it may have been lost in a merge. Conflicts are hard to deal with since I basically have to eyeball it and see if anything changed besides all the indentation I did and usability changes that were done